org.mwc.debrief.legacy/src/Debrief/ReaderWriter/BRT/BRTImporter.java

```diff
@@ -221,13 +273,14 @@ public static TrackWrapper findTrack(final TrackWrapper[] allTracks)
     {
       return allTracks[0];
     }
-    int amountOfBlueTracks = 0, indexOfBlueTrack = 0;
+    int amountOfBlueTracks = 0;
+    int indexOfBlueTrack = 0;
     for (int i = 0; i < allTracks.length; i++)
     {
       if (DebriefColors.BLUE.equals(allTracks[i].getTrackColor()))
       {
         ++amountOfBlueTracks;
```

Comment:
hey, @saulhidalgoaular - I was pleased to spot this next typo.  The tests were all passing, but that was because of the sequence of tracks in the test data 😃

Reply:
Thanks for spotting it :) I will check it more carefully next time.